### PR TITLE
update: octoxlabs integration

### PR DESCRIPTION
**Pull request by ahmetkotan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
demisto/dockerfiles#12797

## Description
Octox Labs Cyber Security Asset Management platform XSoar Integration update.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [x] 6.1.0
- [x] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/OctoxLabs/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### OctoxLabs
+
+- %%UPDATE_RN%%
```
[the remark this answers is not in the document]
Comment (ahmetkotan) [reply Packs/OctoxLabs/ReleaseNotes/1_1_0.md]: Ok, I fixed this. Can you review again?

[the remark this answers is not in the document]
Comment (daryakoval) [reply Packs/OctoxLabs/ReleaseNotes/1_1_0.md]: You just removed this line, you didn't add the real changes to the release notes. 
You should list the new commands you added like in the following [example](https://xsoar.pan.dev/docs/documentation/release-notes#examples).

File: Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.yml [hunk 2]
```diff
@@ -207,12 +207,153 @@ script:
...
+        - contextPath: OctoxLabs.Queries.results.is_temporary
+          description: "Query is temporary"
+          type: Boolean
   isfetch: false
   runonce: false
   script: '-'
   type: python
   subtype: python3
-  dockerimage: demisto/octoxlabs:1.0.0.47334
+  dockerimage: demisto/octoxlabs:1.0.0.49081
```
Comment (daryakoval) [inline Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.yml:356]:
```suggestion
  dockerimage: demisto/octoxlabs:1.0.0.49118
```

File: Packs/OctoxLabs/ReleaseNotes/1_1_0.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### OctoxLabs
+
+- Updated the Docker image to: *demisto/octoxlabs:1.0.0.49081*.
```
Comment (daryakoval) [inline Packs/OctoxLabs/ReleaseNotes/1_1_0.md:6]:
```suggestion
- Updated the Docker image to: *demisto/octoxlabs:1.0.0.49118*.
```

**Review by daryakoval — COMMENTED** [review]
Please update to the latest docker image. (was released after your dockerfiles pr merged).

**Review by daryakoval — APPROVED** [review]

**Pull request merged**